examples/swift/hello_world/DemoFilter.swift

```diff
@@ -17,6 +17,7 @@ struct DemoFilter: ResponseFilter {
   func setResponseFilterCallbacks(_ callbacks: ResponseFilterCallbacks) {}
 
   func onResponseData(_ body: Data, endStream: Bool) -> FilterDataStatus {
+    NSLog("Saw data chunk of length \(body.count)")
```

Comment:
I intend to eventually remove this when we have more direct integration coverage of this functionality in CI, based on the approach @junr03 has proposed with the ValidationFilter (name TBD? ExpectationFilter? RequirementsFilter? SatisfactionFilter.).

Reply:
sg, do you mind leaving a TODO? 

AssertionFilter? So many options!

Reply:
StipulationsFilter.